NitroxClient/GameLogic/PlayerGameObjectManager.cs

```diff
@@ -45,7 +72,30 @@ private GameObject createOtherPlayer(string playerId)
             body.transform.parent.gameObject.GetComponent<Player>().head.shadowCastingMode = UnityEngine.Rendering.ShadowCastingMode.On;
             GameObject bodyCopy = UnityEngine.Object.Instantiate(body);
             body.transform.parent.gameObject.GetComponent<Player>().head.shadowCastingMode = UnityEngine.Rendering.ShadowCastingMode.ShadowsOnly;
-            bodyCopy.transform.Find("player_view").gameObject.GetComponent<ArmsController>().smoothSpeed = 0; //Disables the other character's move animations
+
+            //Get player
+            GameObject playerView = bodyCopy.transform.Find("player_view").gameObject;
```

Comment:
For the future(not this pull request), we'll want to probably tuck this into a clean builder object.